### PR TITLE
testing junit

### PR DESCRIPTION
**Pull request by kaushik-sawant**

### CODE REVIEW

File: core/src/test/java/com/ugamsol/core/models/impl/BannerAreaImplTest.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+class BannerAreaImplTest {
+
+    private final AemContext aemContext = new AemContext();
+    private BannerArea banner;
+
+    @BeforeEach
+    void setUp() {
+
+        //aemContext.addModelsForClasses(BannerAreaImpl.class);
```
Comment (vamshikega) [inline core/src/test/java/com/ugamsol/core/models/impl/BannerAreaImplTest.java]: remove comments

File: core/src/test/java/com/ugamsol/core/models/impl/PriceImplTest.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+@ExtendWith({AemContextExtension.class, MockitoExtension.class})
+class PriceImplTest {
+
+    private final AemContext aemContext = new AemContext();
+    Price price;
+
+    @BeforeEach
+    void setUp() {
+        aemContext.addModelsForClasses(PriceImpl.class);
+        //aemContext.load().json(BannerAreaImpl.class.getResourceAsStream ("com/ugamsol/core/models/impl/Banner.json"),  "/component");
```
Comment (vamshikega) [inline core/src/test/java/com/ugamsol/core/models/impl/PriceImplTest.java]: remove comments

**Review by vamshikega — COMMENTED** [review]
remove comments on the files

File: core/src/test/java/com/ugamsol/core/models/impl/PriceImplTest.java (hunk 2 above)
Comment (kaushik-sawant) [reply core/src/test/java/com/ugamsol/core/models/impl/PriceImplTest.java]: i have removed the comments and commited the code.

File: core/src/test/java/com/ugamsol/core/models/impl/BannerAreaImplTest.java (hunk 1 above)
Comment (kaushik-sawant) [reply core/src/test/java/com/ugamsol/core/models/impl/BannerAreaImplTest.java]: i have removed the comments and comitted again.

**Review by vamshikega — COMMENTED** [review]
approved

**Pull request merged**